.travis/travis-common.inc

```diff
@@ -0,0 +1,68 @@
+#!/bin/bash
+
+set -x
+
+# Global variables to support finalize()
+MMD_OS=
+MMD_RELEASE=
+MMD_TARBALL_PATH=
+
```

Comment:
Optional: consider adding `: ${SCRIPT_DIR:=$(pwd)}` in here, or throw an error if `SCRIPT_DIR` isn't set, since things heavily depend upon it.

Reply:
Hmm, good point. A lot of stuff could break in that case. I'll add it inside the function call, though. I want it to be possible to source the `travis-common.inc` before `$SCRIPT_DIR` is set. It just needs to fail if it's used without it.